core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -722,35 +723,47 @@ public static String getDbVersion() {
     }
     
     public static String getDarwinAuthCheckUrl() {
-        String darwinAuthUrl = properties.getProperty(DARWIN_AUTH_URL);
-        if (darwinAuthUrl == null || darwinAuthUrl.isEmpty()) {
-            return "";
+        String darwinAuthUrl;
+        try{
+            darwinAuthUrl = properties.getProperty(DARWIN_AUTH_URL).trim();            
         }
-        return darwinAuthUrl.trim();
+        catch (NullPointerException e){
```

Comment:
Actually I introduced `org.mskcc.cbio.portal.scripts.TrimmedProperties.java` before for this reason. It is used in a number of places already. Please use that.